### PR TITLE
2016-day-18-part-1

### PR DESCRIPTION
**Pull request by Herrera93**
Created a library to count lights in a two dimensional grid using characters `.` and `#` to indicate if they're on or off.

### CODE REVIEW

**Comment by Herrera93** [comment]
[2015-day-18-part-1](https://trello.com/c/vEqITkac/21-2015-day-18-part-1)

File: tests/day_18.rs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+fn test_day_18_1() {
+    let initial_state: Vec<String> =
+        vec![".#.#.#", "...##.", "#....#", "..#...", "#.#..#", "####.."]
+            .iter()
+            .map(|x| x.to_string())
+            .collect();
+    assert_eq!(4, count_lights(&update_lights_by(&initial_state, 4)));
+}
+
+#[test]
```
Comment (kanekv) [inline tests/day_18.rs]: @Herrera93 is this test fast enough?

Comment (Herrera93) [reply tests/day_18.rs]: I timed it and got 1.044418955 seconds, do you think think this time is fast enough to be a test or should I create a binary instead ?

Comment (kanekv) [reply tests/day_18.rs]: @Herrera93 that's a bit too much, don't need a binary, if there is another test you can just ignore this long one.

Comment (Herrera93) [reply tests/day_18.rs]: @kanekv Cool, then I'll add a `#[ignore]` to it, rebase, and merge.

**Pull request merged**